.circleci/config.yml

```
@@ -27,6 +27,7 @@ commands:
           command: |
             . /opt/conda/etc/profile.d/conda.sh
             conda activate esmvaltool
+            flake8
```

Could you please use
```suggestion
            flake8 -j 4
```
because [automatically detecting the number of CPUs does not work well on CircleCI](https://support.circleci.com/hc/en-us/articles/360005442714-Your-test-tools-are-smart-and-that-s-a-problem-Learn-about-when-optimization-goes-wrong-) machines.

---

OK good find! I think we're both spending a bit too much time on this very technical, minute matter already, must be Friday :grin: